front/app/modules/commercial/idea_custom_fields/index.tsx

```diff
@@ -52,11 +22,19 @@ const configuration: ModuleConfiguration = {
   },
   outlets: {
     'app.containers.Admin.projects.edit': (props) => {
-      return (
-        <RenderOnHideTabCondition project={props.project} phases={props.phases}>
-          <Tab {...props} />
-        </RenderOnHideTabCondition>
+      const { project, phases } = props;
```

Comment:
At the post mortem we did after the previous release of our code, we concluded that we should not release changes to existing code for ideas without tests for that code.

The code changed in this file is responsible to render a tab if necessary. This tab is only rendered for ideation projects/phases. Is there a test that describes this behaviour? If not, we should add it, as agreed at the post mortem.

Reply:
Agreed. It's also this change I pointed out during the chat this morning, which requires testing.

Reply:
I'm currently adding e2e tests for this part of the code coming with a separate MR cc @koendehondt I noticed that you had approved and adding an update here just in case

Reply:
Thanks for the update.

I approved the PR because I like to proceed with the merge to master. In the absence of the test, yesterday Koen G volunteered to perform a manual test. That test was successful. Let's merge the test you are adding later when it is ready.

Reply:
Changes have been finalised [here](https://github.com/CitizenLabDotCo/citizenlab/pull/2658) but I'm okay with merging them after. Thanks for letting me know